test/active_model_serializers/register_jsonapi_renderer_test_isolated.rb

```diff
@@ -12,7 +12,8 @@ class << self
     end
 
     def render_with_jsonapi_renderer
-      author = Author.new(params[:data][:attributes])
+      attributes = params[:data].present? ? params[:data][:attributes] : {}
```

Comment:
this is because when adapter is not set as jsonapi, params are not set.  This is there so that it fails on render and not on trying to set Author object.  Since we use the same controller in case where it should succeed and fail, I wanted to make sure this part doesn't cause a failure.